### PR TITLE
Created doc: Job manager and the JOB statements

### PR DESCRIPTION
**Pull request by randomJoe211**
JOB doc including the new SUBMIT JOB STATS statement

### CODE REVIEW

File: docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
+# Job manager and the JOB statements
+
+The long-term tasks running at the storage layer are called jobs. For example, there are jobs for `COMPACT`, `FLUSH`, and `STATS`. These jobs can be time-consuming if the data size in the graph space is large. The job manager helps you run, show, stop and recover the jobs.
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md:3]: Storage Service. pls unify the terms

File: docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
++------------+
+| 40         |
++------------+
+```
+
+For more information about compact configuration, see Storage Service configuration (doc TODO).
+
+## SUBMIT JOB FLUSH
+
+The `SUBMIT JOB FLUSH` statement writes the RocksDB memfile in memory to the hard disk.
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md:22]: add an eg here?

File: docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+## SUBMIT JOB STATS
+
+The `SUBMIT JOB STATS` statement starts a job that makes the statistics of the current graph space. Once this job succeeds, you can use the `SHOW STATS` statement to list the statistics. For more information, see [SHOW STATS](../7.general-query-statements/6.show/14.show-stats.md).
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md:26]: ditto

File: docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+## SHOW JOB <job_id>
+
+The Meta Service parses a `SUBMIT JOB` request into tasks and assigns them to the storaged processes. The `SHOW JOB <job_id>` statement shows the information about a specific job and all its tasks.
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md:30]: nebula-storaged process

File: docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md (hunk 2 above)
Comment (randomJoe211) [reply docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md:22]: It returns almost the same thing with `SUBMIT JOB COMPACT`, so I think it is not needed.

File: docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md (hunk 4 above)
Comment (randomJoe211) [reply docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md:30]: Oh yes, thank you for pointing this out.

File: docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+| 2              | "storaged1"   | "FINISHED" | 1606544069 | 1606544069 |
++----------------+---------------+------------+------------+------------+
+```
+
+The description of the return message is as follows.
+
+|Column|Description|
+|-|-|
+|Job Id(TaskId)|The first row shows the job ID, and the other rows show the task IDs.|
+|Command(Dest)|The first row shows the command executed, and the other rows show the storaged processes where the task is running.|
```
Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md:54]: the other rows show on which storaged processes the task is running

File: docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md [hunk 6]
```diff
@@ -0,0 +1,135 @@
...
+|Status|Shows the status of the job or task. For more information about job status, see [Job status](#job-status).|
+|Start Time|Shows a timestamp indicating the time when the job or task enters the `RUNNING` phase.|
+|Stop Time|Shows a timestamp indicating the time when the job or task is `FINISHED`, `FAILED`, or `STOPPED`.|
```
Comment (oldLady344) [inline docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md:57]: when the job or task gets `FINISHED`, `FAILED`, or `STOPPED`. OR when the job or task finishes, fails, or stops.

File: docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md (hunk 2 above)
Comment (whitewum) [reply docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md:22]: Give an example. I think.

File: docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md (hunk 3 above)
Comment (whitewum) [reply docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md:26]: Right. An example.

File: docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md [hunk 7]
```diff
@@ -0,0 +1,135 @@
...
+Queue -- running -- finished -- removed
+     \          \                /
+      \          \ -- failed -- /
+       \          \            /
+        \ ---------- stopped -/
+```
+
+## SHOW JOBS
+
+The `SHOW JOBS` statement lists all the jobs that have not expired. The default job expiration time is one week. You can change it by modifying the `job_expired_secs` parameter of the Meta Service.
```
Comment (whitewum) [inline docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md:84]: it should be a TODO to indicate how to change `job_expired_secs` parameter.

Comment (Amber1990Zhang) [inline docs-2.0/3.ngql-guide/18.operation-and-maintenance-statements/4.job-statements.md:84]: have not -> are not

**Review by Amber1990Zhang — DISMISSED** [review]

**Review by whitewum — APPROVED** [review]

**Review by Amber1990Zhang — APPROVED** [review]

**Pull request merged**